docs/src/tool/index.md

```diff
@@ -292,55 +321,56 @@ development API key), and sets `REACH_RPC_TLS_REJECT_UNVERIFIED` to
 Consider this example from the @{seclink("tut-7-rpc")} tutorial:
 
 ```cmd
-$ reach rpc-run python3 -u ./index.py
+$ ./reach rpc-run python3 -u ./index.py
 ```
 
 # {#ref-usage-docker-reset} `reach docker-reset`
 
-You can easily kill and remove all Docker containers by executing
+This command removes all containers and volumes from Docker. If you have any personal containers, those containers will also be removed. Containers can then be restarted in your Docker Images list. It does not uninstall any Docker images.
 
 ```cmd
-$ reach docker-reset
+$ ./reach docker-reset
 ```
 
 This can be a useful thing to try if your Docker containers stop responding to requests or otherwise misbehave, or if you have updated your Reach images (with `reach update`) but those changes are not taking effect.
-This command is a loose approximation of "turning Docker off and on again."
 It will affect all Docker containers on your machine, not just those created by Reach.
 
 # {#ref-usage-upgrade} `reach upgrade`
 
-You can upgrade your Reach installation by executing
+This command downloads the most recent `reachsh/reach-cli` Docker image only.
 
 ```cmd
-$ reach upgrade
+$ ./reach upgrade
 ```
 
-This may change the default version used by `reach` commands.
+This might change the default version used by `reach` commands.
 
 # {#ref-usage-update} `reach update`
 
-You can update the Docker images used by your Reach installation by executing
+This command downloads and installs the most recent versions of the Reach Docker image. It does not start the Docker images once installed.
```

Comment:
I'd undo this change, personally:

- Docker images aren't really _installed_; they're downloaded and then you've got 'em.

- The bit about not starting is sorta implied since we can't predict what _to_ start.

- This line also has a singular vs. plural mismatch.